### ISSUE TITLE
feature: support bentoml.depends.from_url("http://already.deployed-bento.com:3000")

### ISSUE THREAD

**Issue by KimSoungRyoul** [bentoml/BentoML#4745]
### Feature request

~~~Python

@bentoml.service(
    traffic={"timeout": 600},
    workers=8,
resources={"cpu": "1"}
)
class ControlNet:
    # Pass the dependent Service class as an argument
    # controlnet_service = bentoml.depends(SDXLControlNetService)
    controlnet_service = bentoml.depends.from_url(
       url= "http://already.deployed-bento.com:3000",
       input =  Annotated[torch.Tensor, Shape((1, 4)), DType("float32")],
       output =  Annotated[torch.Tensor, Shape((1, 4)), DType("float32")],
    )  

    @bentoml.api
    async def generate(self, image: PIL_Image, params: Params) -> PIL_Image:


~~~

**Comment by frostming** [comment]
Thanks, this will be included in 1.4, the next minor release

**Comment by KimSoungRyoul** [comment]
thx, this feature is already implemented

https://docs.bentoml.com/en/latest/build-with-bentoml/distributed-services.html#depend-on-an-external-deployment

<img width="979" alt="스크린샷 2024-12-22 오전 10 40 52" src="https://github.com/user-attachments/assets/ca4165b2-5494-4218-9ae5-e6986a7814b8" />
